README.md

```
@@ -1,2 +1,197 @@
+| Infra        | Amazon Web Service  | AWS – 클라우드 환경에서 가용성 높은 서비스 가능                         |
+|              | Nginx | Nginx – 리버스 프록시를 통한 SSL/TLS                                 |
+| Mail System  | SMTP                | 이메일 인증을 위한 프로토콜                                             |
+
+
+## 인프라 아키텍처
+
+![](https://velog.velcdn.com/images/dubu1001/post/0cba0d68-d9ee-46d2-8101-31b0ae27056d/image.svg)
+
+## 시스템 아키텍처
+
+![](https://velog.velcdn.com/images/dubu1001/post/5c93fa82-d80b-489b-9d0a-e957991afa8a/image.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)